content/docs/user-guide/components/reference/physx/prismatic-joint.md

```diff
@@ -0,0 +1,65 @@
+---
+linkTitle: PhysX Prismatic Joint
+description: ' Lean to use the Open 3D Engine PhysX Prismatic Joint component. '
+title: PhysX Prismatic Joint Component
+---
+
+
+
+With the **PhysX Prismatic Joint** component, you can create a dynamic prismatic joint that constrains an entity to the joint keeping the same rotation but allowing to move freely along one axis.
+
+## PhysX Prismatic Joint component properties 
+
+![Properties of the PhysX Prismatic Joint component](/images/user-guide/physx/physx/ui-physx-prismatic-joint-component.png)
+
+**Local Position**
+Specify the position of the joint relative to the entity transform.
+
+**Local Rotation**
+Specify the rotation of the joint relative to the entity transform.
+
+**Lead Entity**
+Specify the parent entity that will drive the joint.
+
+**Breakable**
+When enabled, the joint will break if sufficient force is applied. Enabling **Breakable** exposes the **Maximum Force** and **Maximum Torque** properties.
+**PhysX Dynamic Rigid Body** components that have their **Compute Mass** property enabled might have very large mass values. If the entity containing the joint component or the leader entity have their **Compute Mass** property enabled, the **Maximum Force** and **Maximum Torque** properties might require very high values to resist breaking.
+
+**Maximum Force**
+When **Breakable** is enabled, specify the maximum force the joint can sustain before breaking. Valid values range from **0.01** to **Infinity**.
+
+**Maximum Torque**
+When **Breakable** is enabled, specify the maximum torque the joint can sustain before breaking. Valid values range from **0.01** to **Infinity**.
+
+**Display Setup in Viewport**
+When enabled, two planes are displayed showing the limits of the joint. Red and green planes show the **Lower linear limit** and **Upper linear limit**. A line displays the connection between the joint and the **Follower Entity**.
+
+**Select Lead on Snap**
+When enabled, snapping the joint to an entity in component mode will set the entity as the **Lead Entity**. The entity containing the joint component is excluded from this operation.
```

Comment:
question: Does the prismatic joint have a component mode?

Reply:
Good point... No. But it does have the "Select Lead on Snap" properly because it's part of the common properties of all joints, so it's not trivial to hide it just for Prismatic joints. I guess it was always the plan to add component mode at some point but it never happened, Prismatic joint was added in a rush as a customer needed it quick.

Any advise on how to proceed? Changing the code now is a very last moment change, but not putting in the documentation doesn't align with the list of properties.

Maybe for now we can add a note at the end of that property saying "At the moment component mode is not supported in Prismatic joints" or something like that. Thoughts?

Reply:
Okay, for now I think we should remove any mention of Component Mode otherwise it'll be confusing if it's not there and we can update it when/if it is added. No code changes are required, we just need to make sure the docs correctly reflect reality.

Reply:
Discussed offline, for now we'll just add a note in that property for prismatic joints indicating that component mode is not supported in prismatic joints at the moment.